docs/source/en/quantization/bitsandbytes.md

```diff
@@ -13,88 +13,208 @@ specific language governing permissions and limitations under the License.
 
 # bitsandbytes
 
-[bitsandbytes](https://huggingface.co/docs/bitsandbytes/index) is the easiest option for quantizing a model to 8 and 4-bit. 8-bit quantization multiplies outliers in fp16 with non-outliers in int8, converts the non-outlier values back to fp16, and then adds them together to return the weights in fp16. This reduces the degradative effect outlier values have on a model's performance.
+[bitsandbytes](https://huggingface.co/docs/bitsandbytes/index) is the easiest option for quantizing
+a model to 8 and 4-bit. 8-bit quantization multiplies outliers in fp16 with non-outliers in int8,
+converts the non-outlier values back to fp16, and then adds them together to return the weights in
+fp16. This reduces the degradative effect outlier values have on a model's performance.
 
-4-bit quantization compresses a model even further, and it is commonly used with [QLoRA](https://hf.co/papers/2305.14314) to finetune quantized LLMs.
+4-bit quantization compresses a model even further, and it is commonly used with
+[QLoRA](https://hf.co/papers/2305.14314) to finetune quantized LLMs.
 
+This guide demonstrates how quantization can enable running
+[FLUX.1-dev](https://huggingface.co/black-forest-labs/FLUX.1-dev)
+on less than 16GB of VRAM and even on a free Google
+Colab instance.
+
+![comparison image](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/diffusers/quant-bnb/comparison.png)
 
 To use bitsandbytes, make sure you have the following libraries installed:
 
 ```bash
 pip install diffusers transformers accelerate bitsandbytes -U
 ```
 
-Now you can quantize a model by passing a [`BitsAndBytesConfig`] to [`~ModelMixin.from_pretrained`]. This works for any model in any modality, as long as it supports loading with [Accelerate](https://hf.co/docs/accelerate/index) and contains `torch.nn.Linear` layers.
+Now you can quantize a model by passing a [`BitsAndBytesConfig`] to [`~ModelMixin.from_pretrained`].
+This works for any model in any modality, as long as it supports loading with
+[Accelerate](https://hf.co/docs/accelerate/index) and contains `torch.nn.Linear` layers.
```

Comment:
Seems like a lot of the changes are related to breaking of long lines? If possible, could we avoid those?